bosh-setup/azuredeploy.json

```diff
@@ -2,149 +2,114 @@
   "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
   "contentVersion": "1.0.0.0",
   "parameters": {
-    "newStorageAccountName": {
-      "type": "string",
-      "metadata": {
-        "description": "Unique DNS Name for the Storage Account where the Virtual Machine's disks will be placed. It must be between 3 and 24 characters in length and use numbers and lower-case letters only."
-      }
-    },
-    "virtualNetworkName": {
-      "type": "string",
-      "defaultValue": "boshvnet-crp",
-      "metadata": {
-        "description": "name of the virtual network"
-      }
-    },
-    "subnetNameForBosh": {
-      "type": "string",
-      "defaultValue": "Bosh",
-      "metadata": {
-        "description": "name of the subnet for Bosh"
-      }
-    },
-    "subnetNameForCloudFoundry": {
-      "type": "string",
-      "defaultValue": "CloudFoundry",
-      "metadata": {
-        "description": "name of the subnet for CloudFoundy"
-      }
-    },
     "vmName": {
```

Comment:
@singhkay 
Will `concat(toLower(parameters('vmName'), uniqueString(resourceGroup().id))` work for `domainNameLabel`?